src/legacy/core_plugins/kibana/public/visualize/legacy.ts

```diff
@@ -27,9 +27,7 @@ import { plugin } from './index';
   const instance = plugin({} as PluginInitializerContext);
   instance.setup(npSetup.core, {
     ...npSetup.plugins,
-    __LEGACY: {
-      legacyChrome,
-    },
+    npData: npSetup.plugins.data,
```

Comment:
Why the redundant `npData: npSetup.plugins.data` ? why not just `instance.setup(npSetup.core, npSetup.plugins)` and access the `data` plugin with it's proper `data` name?

Reply:
@pgayvallet I will clean this up with https://github.com/elastic/kibana/pull/57331